### PR TITLE
CLDR-17566 converting index acknowledgments downloads

### PR DESCRIPTION
**Pull request by chpy04**
[CLDR-17566](https://unicode-org.atlassian.net/browse/CLDR-17566)

Part of the [CLDR --> GFM conversion project](https://docs.google.com/document/d/1NoQX0zqSYqU4CUuNijTWKQaphE4SCuHl6Bej2C4mb58/edit)

Converting https://cldr.unicode.org/development/index,
https://cldr.unicode.org/index/acknowledgments,
https://cldr.unicode.org/index/downloads,
https://cldr.unicode.org/index/cldr-spec,
https://cldr.unicode.org/index/corrigenda,
https://cldr.unicode.org/index/draft-schedule,
https://cldr.unicode.org/index/json-format-data,
https://cldr.unicode.org/index/language-support-levels,
https://cldr.unicode.org/index/requesting-additionsupdates-to-cldr-languagepopulation-data,
https://cldr.unicode.org/index/cldr-presentations,
https://cldr.unicode.org/index/locale-coverage

View text diffs for:

[index](https://docs.google.com/document/d/1WGQyZAYDvEmn5CkXEWvBiZATz0Qxd7KdLtOtx78h9_E/edit?usp=sharing)
[acknowledgments](https://docs.google.com/document/d/1suA1SKu5VayyM9CJ_neTlZxsqWp5yd-OklkVyXbE8vg/edit?usp=sharing)
[downloads](https://docs.google.com/document/d/1l9WREUFx5Cc33lotBEkQU5eCloTuuCyRIOsf-xhNylo/edit?usp=sharing)
[cldr-spec](https://docs.google.com/document/d/1lx1gOAzINGoAiWUcXkv8eJtRslNRm1n51fcG-BvJpp4/edit?usp=sharing)
[corrigenda](https://docs.google.com/document/d/1j-8muoE07wg6KPKTrEpfUvPvFpuIawkNPR0xSvmsVWc/edit?usp=sharing)
[draft-schedule](https://docs.google.com/document/d/12F5vKaOeKnf3LC6JNCCO2TG9IaaBc48gf0cCV9J_YB8/edit?usp=sharing)
[json-format-data](https://docs.google.com/document/d/1GNgUPYNU9Bn8BQCvT16gDnY3oR8LnsDQ5jldPHDmJKg/edit?usp=sharing)
[language-support-levels](https://docs.google.com/document/d/1zPddD3Ae_L1K2GWrxE-5UffB9rqaq2Xa1KzlTrgORQ4/edit?usp=sharing)
[requesting-additionsupdates-to-cldr-languagepopulation-data](https://docs.google.com/document/d/1SXwZ4QtVCusGU8ciKBW1daCuY0YEXZcZyrG5h2MNfmI/edit?usp=sharing)
[cldr-presentations](https://docs.google.com/document/d/1nqZmObHzu_MHqI6AX8oayHdNOQl7XLfZDjchMTp5CYI/edit?usp=sharing)
[locale-coverage](https://docs.google.com/document/d/1iJZRBVprtTE-ZZmcjnryX8O3fSi7KM1YDz4dULYTiFw/edit?usp=sharing)
ALLOW_MANY_COMMITS=true

[CLDR-17566]: https://unicode-org.atlassian.net/browse/CLDR-17566?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/site/index/language-support-levels [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
```
Comment (macchiati) [inline docs/site/index/language-support-levels:1]: There appears to be something off in this file. There is a warning symbol at the end, and I can't view as HTML

**Review by macchiati — CHANGES_REQUESTED** [review]
One issue, otherwise looking good.

File: docs/site/index/language-support-levels (hunk 1 above)
Comment (chpy04) [reply docs/site/index/language-support-levels]: Oops, it wasn't a markdown file. Should be good now.

**Review by macchiati — APPROVED** [review]

**Comment by macchiati** [comment]
Notes for the future: 

1. You need to have each commit start with precisely the characters below. We can fix this by squashing using a web tool, but it is a bit of a pain

CLDR<hyphen><ticket-number><space>....

Eg

CLDR-17566 converting index acknowledgments downloads

2. I sent you a note about the text diffs.
3. Probably best to do this in chunks of up to 6(ish) files. Makes it easier to split reviews.

**Comment by macchiati** [comment]
Ah, I remembered. You will have to squash this because I don't have access to your branch.

**Comment by chpy04** [comment]
Ok let me know if that is good now. Also on the next batch I will try out using the github diff with a temp folder, so we can see how that goes.

**Comment by macchiati** [comment]
Hmmm. That's not working. You might have to create another PR with these files.

**Comment by srl295** [comment]
> Hmmm. That's not working. You might have to create another PR with these files.

Just needed the PR title fixed. I fixed it.

**Review by macchiati — COMMENTED** [review]
The PR includes 2 class files CheckDates$... Once you remove those, I can approve and merge.

**Review by macchiati — APPROVED** [review]

**Comment by srl295** [comment]
@chpy04 is this one obsolete? or does it need squash?

**Review by srl295 — APPROVED** [review]

**Comment by chpy04** [comment]
@srl295 needed squash, should be good now

**Pull request merged**